cli/README.md

```diff
@@ -9,5 +9,28 @@
 make cli
 ./rill --help 
 ```
+2. To install it via homebrew
+```
+brew install rilldata/rill-developer/rill
+rill --help
+```
+
+3. You can also try our example:
+    1. List of available examples
+    ```
+    rill init list
+    ```
+    2. Initialize the example project, default to `default`, default directory `.`
+    ```
+    rill init --example <example project name> --dir <directory to migrate example project>
```

Comment:
Is this upto date?

Reply:
If you mean it this complete, then no, Just added the some of commands which are working as per new logic. Still CLI is in progress. For few commands logic is not implemented yet.